src/Nri/Ui/Text/V1.elm

```diff
@@ -150,6 +155,34 @@ classString classes =
     String.join " " (List.map (identifierToString namespace) classes)
 
 
+{-| User-generated text.
+-}
+ugMediumBody : List (Html.Styled.Html msg) -> Html.Styled.Html msg
```

Comment:
For the QW grading page we are using `white-space: pre-line` to display newlines.
Does that make sense for your use cases? I'm thinking it's a reasonable default.

Reply:
Yeah that makes sense to me! @bendansby is there any case where user generated text **shouldn't** be whitespace preline?

Reply:
Nothing comes to mindx

Reply:
Sweet! added :)